src/codeu/chat/common/ServerInfo.java

```diff
@@ -0,0 +1,29 @@
+package codeu.chat.common;
+
+import codeu.chat.util.Time;
+
+public final class ServerInfo {
```

Comment:
nit: remove extra blank lines in the constructor, add 1 blank line between methods.